public/admin/config.yml

```
@@ -59,7 +59,13 @@ collections:
         required: false
         i18n: true
         fields:
-          - { label: "Image", name: "src", widget: "image", i18n: true }
+          - {
+              label: "Image",
+              name: "src",
+              widget: "image",
+              i18n: true,
+              hint: "Please keep file size under 1 megabyte.",
```

I think people are used to seeing MB.
```suggestion
              hint: "Please keep file size under 1MB.",
```